controllers/goharbor/harbor/harbor.go

```diff
@@ -0,0 +1,81 @@
+package harbor
+
+import (
+	"context"
+	"time"
+
+	certv1 "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha2"
+	"github.com/ovh/configstore"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	netv1 "k8s.io/api/networking/v1beta1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+
+	goharborv1alpha2 "github.com/goharbor/harbor-operator/apis/goharbor.io/v1alpha2"
+	"github.com/goharbor/harbor-operator/pkg/config"
+	commonCtrl "github.com/goharbor/harbor-operator/pkg/controller"
```

Comment:
I did not find this package at both this PR and the current master branch. What I'm missing?

Reply:
I use the PR review tool of VSCODE.

Reply:
The package is on this `1.0-rc` branch: <https://github.com/goharbor/harbor-operator/tree/1.0-rc/pkg/controller>
VSCode should be able to use the local directory to find it since [`go.mod`](https://github.com/goharbor/harbor-operator/blob/1.0-rc/go.mod#L1) declare `module github.com/goharbor/harbor-operator`.
I do not know why it is broken.
Did you fix it since then?